docs/contributing/Tree-hygiene.md

```diff
@@ -301,12 +307,6 @@ the repository yet, then wait for one of the project maintainers to submit it fo
 
 If you do have access, add the "autosubmit" label. A bot will land the PR for you.
 
-
-### Squashing commits
-
-When you squash commits, by default, GitHub will concatenate all your commit messages to form a unified commit message.  This often yields an overly verbose commit message with many unhelpful entries (e.g. "fix typo").  Please double-check (and hand-edit if necessary) your commit message before merging such that the message contains a helpful description of the overall change.
```

Comment:
Oh yeah this is funny, I thought the PR description ends up being the commit message.